client/app/bundles/course/assessment/submission/components/answers/ForumPostResponse/CardTitle.jsx

```diff
@@ -20,8 +21,12 @@ const styles = {
 
 const CardTitle = ({ type, title }) => (
   <div style={styles.cardTitle}>
-    <div style={styles.typeLabel}>{type}</div>
-    <div>{title}</div>
+    <div style={styles.typeLabel}>
```

Comment:
roger that. Though, the transition to tailwind render some styles to be unable to be applied directly.
- in case of maxWidth: 600, there is no direct translation to it since `max-w-xxx` does not have the direct one equal to this, so I changed it to `max-w-5xl` instead, which is the closest one
- also, `overflow-hidden` and `whitespace-nowrap` seems does not work properly with this. If I set the max width into `max-w-4xl`, it will cut the text quite a lot and leaves lots of spaces between the title and the markdown button. On the other hand, setting max-width to `max-w-5xl` will render the border to be not uniform across the card

Using `max-w-5xl`, `overflow-hidden`, and `whitespace-nowrap`
![Screenshot 2024-01-02 at 11 57 05 AM](https://github.com/Coursemology/coursemology2/assets/16359075/679174c2-c513-44d3-b29f-6df3abd26b7b)

Using `max-w-4xl`, `overflow-hidden`, and `whitespace-nowrap`
![Screenshot 2024-01-02 at 11 57 33 AM](https://github.com/Coursemology/coursemology2/assets/16359075/1708d70a-8a7d-4cc9-8e89-c1b9bd90587a)

Using `max-w-5xl` only (having `overflow-hidden` without `whitespace-nowrap` will not have any effect)
![Screenshot 2024-01-02 at 11 57 59 AM](https://github.com/Coursemology/coursemology2/assets/16359075/03d566c9-686d-45ac-8468-b7ffcdd37cc2)

Personally I think the last one is the best among the other 2, but please advise on this. Thanks

Reply:
How about `max-w-[123px]`?

Reply:
updated.. I was trying to avoid using pixel previously, but seems like it's quite unavoidable here
I used 600px and it turned out fine